src/main/java/com/parsing/exception/CropImageException.java

```diff
@@ -0,0 +1,7 @@
+package com.parsing.exception;
+
+public class CropImageException extends RuntimeException {
+    public CropImageException(String message, Throwable cause) {
+        super(String.format("method \"CropImage.cropImage\" fail. %s ", message), cause);
```

Comment:
a path where an exception was thrown is in a cause. Don't add it to the message.

Reply:
done